src/net_processing.cpp

```diff
@@ -398,6 +398,10 @@ struct Peer {
     /** Peer selected us as (compact blocks) high-bandwidth peer (BIP152). */
     std::atomic<bool> m_bip152_highbandwidth_from{false};
 
+    /** Time offset computed during the version handshake based on the
+     * timestamp the peer send in the initial version message. */
```

Comment:
nit: sent; also maybe drop "initial", it's not like they send multiple version msgs.